### ISSUE TITLE
Plugin allows creation of usernames containing characters that are not allowed in system.yaml

### ISSUE THREAD

**Issue by boettges** [david-szabo97/grav-plugin-admin-addon-user-manager#21]
Hello,
I found that by default system.yaml disallows the usage of the "." (dot) character in admin-usernames during the initial setup of Grav.

This is due to:
`username_regex: '^[a-z0-9_-]{3,16}$'`
in grav/system/config/system.yaml

However, if a user is subsequently created with the user manager addon usernames that contain the dot-character will be created without any issue.

If the user manager addon hooks into grav's default user creation function, why is there a different set of characters allowed?

Thanks a lot in advance

**Comment by david-szabo97** [comment]
Hi there,

That regex is only used at the initial admin user creation. Grav admin
doesn't use it anywhere else. So I think this should be fixed on Grav Admin
side rather than in this plugin.

I will take a deeper look at it next week. (I am not home this weekend)

**Comment by boettges** [comment]
No worries 👌
Just stumbled upon this issue.

If I should suggest something, I would let this user manager plugin hook into the same check as the admin user. 

I tried to create users, called "test/test", "test%test" and "test$test" and ran into all kinds of issues.

I already sent a pull request to the main branch of grav to include the dot-character into the regex-check for admin-users, as I experienced no issues with dot-containing-usernames.
